### PR TITLE
Merge 2.3 into master

### PR DESCRIPTION
**Pull request by rouliane**
**Description (for Contributor and Core Developer)**

**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | Todo
| Added legacy Behats               | Todo
| Added acceptance tests            | Todo
| Added integration tests           | Todo
| Changelog updated                 | Todo
| Review and 2 GTM                  | Todo
| Micro Demo to the PO (Story only) | Todo
| Migration script                  | -
| Tech Doc                          | -

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

File: src/Pim/Bundle/CatalogBundle/Version.php [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+namespace Pim\Bundle\CatalogBundle;
+
+/**
+ * PIM Version
+ *
+ * @author    Nicolas Dupont <nicolas@akeneo.com>
+ * @copyright 2014 Akeneo SAS (http://www.akeneo.com)
+ * @license   http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+class Version
```
Comment (rouliane) [inline src/Pim/Bundle/CatalogBundle/Version.php]: To remove (and update Akeneo\Platform\CommunityVersion instead)

File: src/Pim/Bundle/CatalogBundle/Doctrine/ORM/Query/GetAssociatedProductCodesByProductFromDB.php [hunk 2]
```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Pim\Bundle\CatalogBundle\Doctrine\ORM\Query;
+
+use Doctrine\ORM\EntityManagerInterface;
+use Pim\Component\Catalog\Model\AssociationInterface;
+use Pim\Component\Catalog\Query\AssociatedProduct\GetAssociatedProductCodesByProduct;
+
+class GetAssociatedProductCodesByProductFromDB implements GetAssociatedProductCodesByProduct
```
Comment (rouliane) [inline src/Pim/Bundle/CatalogBundle/Doctrine/ORM/Query/GetAssociatedProductCodesByProductFromDB.php]: Move to Akeneo\Pim\Enrichment\Bundle\Doctrine\ORM\Query (and its integration test too)

File: src/Pim/Component/Catalog/Query/AssociatedProduct/GetAssociatedProductCodesByProduct.php [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+namespace Pim\Component\Catalog\Query\AssociatedProduct;
+
+use Akeneo\Pim\Enrichment\Component\Product\Model\AssociationInterface;
+
+/**
+ * @copyright 2018 Akeneo SAS (http://www.akeneo.com)
+ * @license   http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+interface GetAssociatedProductCodesByProduct
```
Comment (rouliane) [inline src/Pim/Component/Catalog/Query/AssociatedProduct/GetAssociatedProductCodesByProduct.php]: Move this class and its directory to Akeneo\Pim\Enrichment\Component\Product\Query

File: .ci/Jenkinsfile [hunk 4]
```diff
@@ -1,5 +1,5 @@
 #!groovy
-@Library("k8s-utils@v2.0.0")
+@Library("k8s-utils@override")
```
Comment (Doodoune) [inline .ci/Jenkinsfile]: Don't forget to remove it before merge

Comment (damien-carcel) [reply .ci/Jenkinsfile]: In fact, we will merge it like that, and make a bunch of PRs to set the v2.0.1 once released.

**Pull request merged**